rosbag2_snapshot/src/snapshotter.cpp

```diff
@@ -791,7 +961,7 @@ void SnapshotterClient::setSnapshotterClientOptions(const SnapshotterClientOptio
     if (req->filename.empty()) {
       req->filename = "./";
     }
-    std::filesystem::path p(std::filesystem::absolute(req->filename));
+    std::experimental::filesystem::path p(std::experimental::filesystem::absolute(req->filename));
```

Comment:
This is where the namespace corresponding to #include <filesystem> is used in the code.